sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -1092,3 +1094,50 @@ def _from_generated(cls, generated):
             )
             return scope
         return None
+
+
+class DelimitedTextConfiguration(GenDelimitedTextConfiguration):
+    """Defines the input or output delimited (CSV) serialization for a blob quick query request.
+
+    :keyword str column_separator: Required. column separator
+    :keyword str field_quote: Required. field quote
+    :keyword str record_separator: Required. record separator
+    :keyword str escape_char: Required. escape char
+    :keyword bool headers_present: Required. has headers
+    """
+    def __init__(self, **kwargs):
+        field_quote = kwargs.pop('field_quote', '"')
+        escape_char = kwargs.pop('escape_char', "")
+        super(DelimitedTextConfiguration, self).__init__(field_quote=field_quote,
+                                                         escape_char=escape_char,
+                                                         **kwargs)
+
+
+class JsonTextConfiguration(GenJsonTextConfiguration):
+    """Defines the input or output JSON serialization for a blob quick query request.
+
+    :keyword str record_separator: Required. record separator
```

Comment:
Docstring says there's a required parameter?

Reply:
this class should be removed